protocol.cpp

```diff
@@ -631,11 +631,6 @@ bool redis_protocol::format_arbitrary_command(arbitrary_command &cmd) {
 
         // check arg type
         if (current_arg->data.find(KEY_PLACEHOLDER) != std::string::npos) {
```

Comment:
do we want to allow two or more `__key__`, for ex: "--command "SET bla__key__bla__key__bla value"?

Reply:
@yaacov I believe we need it we should address it on another PR.

Reply:
Agree, but let's verify it, to avoid confusion now that we allow more than just __key__